src/Chapter1.hs

```diff
@@ -653,7 +670,11 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | (abs n) < 10 = (abs n)
+  | otherwise = firstDigit(div (abs n) 10) 
+-- firstDigit = if n < 10 then n else firstDigit(div n 10) - Variable not in scope... why?
```

Comment:
I see that you didn't specify that this function is taking `n` as an argument

```suggestion
-- firstDigit n = if n < 10 then n else firstDigit(div n 10) - Variable not in scope... why?
```
This caused the error 🙂 
This solution is correct as well, though you would also need to use `abs` as in the one above 👍🏼

Reply:
🤦‍♀️ Oh my... how did I miss that? Thank you 
And yes, I would need to add abs. I only used abs in the solutions after running the test and some of them were failing. That's when I realised I should add use abs.